app/controllers/api/base_controller.rb

```diff
@@ -59,6 +59,22 @@ def index
       render_collection(@req.subject, res, opts)
     end
 
+    def create
+      if @req.resources.any? { |r| parse_id(r, @req.subject.to_sym) }
+        raise BadRequestError, "Resource id or href should not be specified for creating a new #{@req.subject}"
+      end
+
+      results = @req.resources.collect do |r|
+        if @req.subcollection?
+          send("#{@req.subject}_create_resource", parent_resource_obj, @req.subject.to_sym, nil, r)
```

Comment:
Continue using `target_resource_method` instead of interpolating yourself here; if we want to remove that it's out of scope for what you're doing here.